release_workspace.js

```diff
@@ -0,0 +1,294 @@
+const { execSync, exec } = require("child_process");
+const { Command } = require("commander");
+const fs = require("fs");
+const path = require("path");
+const { spawn } = require("child_process");
+const readline = require("readline");
+
+/**
+ * Finds all workspaces in the monorepo and returns an array of objects.
+ * Each object in the return value contains the relative path to the workspace
+ * directory, along with the full package.json file contents.
+ * 
+ * @returns {Array<{ dir: string, packageJSON: Record<string, any>}>}
+ */
+function getAllWorkspaces() {
+  const possibleWorkspaceDirectories = ["./libs/*", "./langchain", "./langchain-core"];
+  const allWorkspaces = possibleWorkspaceDirectories.flatMap((workspaceDirectory) => {
+    if (workspaceDirectory.endsWith("*")) {
+      // List all folders inside directory, require, and return the package.json.
+      const allDirs = fs.readdirSync(path.join(process.cwd(), workspaceDirectory.replace("*", "")));
+      const subDirs = allDirs.map((dir) => {
+        return {
+          dir: `${workspaceDirectory.replace("*", "")}${dir}`,
+          packageJSON: require(path.join(process.cwd(), `${workspaceDirectory.replace("*", "")}${dir}`, "package.json"))
+        }
+      });
+      return subDirs;
+    }
+    const packageJSON = require(path.join(process.cwd(), workspaceDirectory, "package.json"));
+    return {
+      dir: workspaceDirectory,
+      packageJSON,
+    };
+  });
+  return allWorkspaces;
+}
+
+/**
+ * Increments the last numeric character in a version string by 1.
+ * If the last character is not numeric, it searches backwards
+ * to find the last numeric character to increment.
+ * 
+ * @param {string} version 
+ * @returns {string} The new version
+ */
+function bumpVersion(version) {
```

Comment:
I think `release-it` does this by default if configured correctly?

Reply:
Oh, this is for bumping deps

Reply:
yes, unless we want to pass a custom version to change to, in that case you need to specify the version directly. Granted, not sure if this ever comes up I just assumed it might. If not, lmk and I can revert the changes which prevents `release-it` from bumping the version on its own.